documentation/index.json

```diff
@@ -6,26 +6,26 @@
             "default_tab": "yes",
             "subitems": [
                 {
-                    "title": "Getting Started",
-                    "description": "This is the stuff about how to get started",
+                    "title": "Getting started",
```

Comment:
I think you need to have a discussion with Alasdair about capitalisation in titles. At present, the documentation repo capitalises each word, not just the first word: this was by design per Alasdair.

Reply:
Mate, I sit next to Alasdair. I'm his boss. And, remarkably enough, we talk to each other about the work we do. Happily, we are both fully up to speed with how things work here.

Could you _please_ stop making this sort of stylistic/editorial contribution? We're not looking for this sort of commentary from outside Raspberry Pi; it wastes everybody's time in task-switching, and we have context and background that aren't available to people outside the organisation. Think you've got a technical fix? Fine: go ahead and make a PR. But please stop with the editorial "help"; it's not actually at all helpful, and as I've said to you before, we have a workflow in place that this just interrupts.

Reply:
Sorry Liz. I must apologise: I've clearly got this rather wrong. In the future if I contribute I will ensure I confine myself solely to the technical content.

Reply:
Thanks - no hard feelings!